starkbank/transfer/rules/__rule.py

```diff
@@ -0,0 +1,18 @@
+from starkcore.utils.subresource import SubResource
+
+
+class Rule(SubResource):
+
+    """# Transfer.Rule object
+    The Transfer.Rule object modify the behavior of Transfer objects created in your Workspace.
```

Comment:
The Transfer.Rule object modifies the behavior of Transfer objects when passed as an argument upon their creation.